.github/workflows/ci.yml

```diff
@@ -146,20 +146,14 @@ jobs:
         name: Prepare LD_LIBRARY_PATH (Ubuntu only)
         run: echo LD_LIBRARY_PATH=${pythonLocation}/lib >> $GITHUB_ENV
 
-      - name: Prepare workflow settings
-        id: settings
-        shell: bash
-        run: |
-          echo "::set-output name=all_additive_features::$(make list_all_additive_features)"
-
       - if: matrix.msrv == 'MSRV'
         name: Prepare minimal package versions (MSRV only)
         run: |
           cargo update -p indexmap --precise 1.6.2
           cargo update -p hashbrown:0.11.2 --precise 0.9.1
 
       - name: Build docs
-        run: cargo doc --no-deps --no-default-features --features "${{ steps.settings.outputs.all_additive_features }}"
+        run: cargo doc --no-deps --no-default-features --features full
```

Comment:
Should this become `cargo pyo3_doc` ?

Reply:
Actually, would you be OK with removing this step? Building docs is also tested in `guide.yml`.

Reply:
Hmmm, I do think there's value in checking the docs build on all each of windows, Linux and macos. Probably doesn't need to be run for each Python variant. Could make it 3.10-only?